contextid.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"context"
+	"net/http"
+)
+
+// AddContextID wraps a http.Handler to add a monotonically increasing
```

Comment:
I don't think "monotonically" is the right word here:
>  monotonic increasing: (mathematics, of a function) always increasing or remaining constant, and never decreasing; contrast this with strictly increasing ([wiktionary](https://en.wiktionary.org/wiki/monotonic_increasing))
IDs should be strictly increasing, right?

Reply:
Done. Changed to "strictly increasing" locally. Will push the fix after further reviews come in.

Reply:
Fixed in 8ecf68b7141491dcacaa83cc64922fb73463530a. Will rebase-squash prior to merging.

Reply:
actually I still think this is not the right term (sorry should have noticed earlier).
It's only used for continuous and not discrete functions, but I might be wrong here...
Doesn't matter all that much, you communicate clearly what it is: an "autoincrement", "serial" kinda thing from the DB perspective.

Reply:
The definition “for all _x_ and _y_ such that _x_ < _y_ one has _f_(_x_) < _f_(_y_)” applies equally well to discrete functions.

Reply:
💥  :)